### PR TITLE
experiment(plugin-image): experiment with direct upload

### PR DESCRIPTION
**Pull request by elbotho**

### CODE REVIEW

File: packages/editor/src/plugins/rows/editor-renderer.tsx [hunk 1]
```diff
@@ -149,11 +149,11 @@ export function EditorRowRenderer({
         case NativeTypes.URL: {
           const urls: string[] = monitor.getItem<{ urls: string[] }>().urls
           const text = urls[0]
-          plugins.find(({ type, plugin }) => {
+          plugins.find(async ({ type, plugin }) => {
```
Comment (Entkenntnis) [inline packages/editor/src/plugins/rows/editor-renderer.tsx]: not 100% if `plugins.find` can properly handle the aync and return the awaited value (most likely not - but we don't actually use the return, so it's fine here)

I would suggest adding a comment here.

**Review by Entkenntnis — APPROVED** [review]

File: packages/editor/src/plugins/rows/editor-renderer.tsx (hunk 1 above)
Comment (elbotho) [reply packages/editor/src/plugins/rows/editor-renderer.tsx]: good catch!

Comment (elbotho) [reply packages/editor/src/plugins/rows/editor-renderer.tsx]: what do you think of this refactor?
https://github.com/serlo/frontend/pull/3293/commits/c1bb107a0d4b713c4c6f8fffe0faeadb3d062845

Comment (Entkenntnis) [reply packages/editor/src/plugins/rows/editor-renderer.tsx]: thank you, the refactor makes the change of behaviour clearer

**Pull request merged**